src/DynamoCoreWpf/Views/Core/DynamoView.xaml

```diff
@@ -1516,7 +1516,7 @@
                     VerticalAlignment="Top"
                     Margin="0,310,0,0">
             <Grid Background="#353535"
-                  Width="100"
+                  Width="Auto"
```

Comment:
Seems this is the only changed needed to fix the left side?

Reply:
Yes. No further gymnastics required on that side.